components/bl-stepper-component/styles/index.less

```diff
@@ -0,0 +1,190 @@
+// you can use LESS variables from the current Theme and app extensions of the UI-Container
+@bl-customComponent-stepper-themeColor: @themePrimary;
+@bl-customComponent-stepper-backgroundColor: @appBackgroundColor;
+@bl-customComponent-stepper-textColor: @appTextColor;
+
+@bl-customComponent-stepper-transition: 0.3s;
+
+@bl-customComponent-stepper-iconSize: 24px;
+@bl-customComponent-stepper-iconBorderRadius: 50%;
+@bl-customComponent-stepper-itemWidth: 100%;
+@bl-customComponent-stepper-lineWidth: 100%;
+@bl-customComponent-stepper-lineColor: rgb(117, 117, 117);
+@bl-customComponent-stepper-lineBorderWidth: 1px;
+
+@bl-customComponent-stepper-alternative-label-titleTextMarginTop: 16px;
+
+@bl-customComponent-stepper-customized-lineBorderWidth: 3px;
+@bl-customComponent-stepper-customized-iconSize: 18px;
+@bl-customComponent-stepper-customized-iconMarkSize: 8px;
+
+.stepper {
```

Comment:
change root class to `.bl-customComponent-stepper`

and move everyone else inside it